backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java

```diff
@@ -39,26 +39,55 @@ void WhenListIsEmptyReturnEmptyList() throws Exception {
     void WhenEntryIsAddedReturnAddedEntry() throws Exception {
         //When
         mockMvc.perform(
-                MockMvcRequestBuilders.post("/api/entries")
-                        .contentType(MediaType.APPLICATION_JSON)
-                        .content("{\n" +
-                                "    \"title\": \"testTitle\",\n" +
-                                "    \"description\": \"testDescription\",\n" +
-                                "    \"date\": \"2023-12-03\",\n" +
-                                "    \"amount\": 34,\n" +
-                                "    \"category\": \"INCOME\",\n" +
-                                "    \"interval\": \"MONTHLY\"\n" +
-                                "}")
-        )
-        //Then
+                        MockMvcRequestBuilders.post("/api/entries")
+                                .contentType(MediaType.APPLICATION_JSON)
+                                .content("{\n" +
+                                        "    \"title\": \"testTitle\",\n" +
```

Comment:
`
@Autowired
Objectmapper objectMapper;

String requestBody = objectMapper.writeValueAsString(your-Class-Instance-variable);

 .content(requestBody);
`

Reply:
Thank you for this suggestion. This is much better;)